operations/experimental/README.md

```diff
@@ -9,12 +9,113 @@ This is the README for Experimental Ops-files. To learn more about `cf-deploymen
+                                                    |
+                                                    |
+                                                    v
+                                        +------------------------+
+                                        |                        |
+                                        |   RelInt cuts a minor  |
+                                        |   release.             |
+                                        |         (•‿•)          |
+                                        +------------------------+
+```
+
+## Promoting an experimental ops file - Symlink Option
+
+_Please do not remove the file or it's content._
+```
++------------------------+             +-------------------------+              +----------------------------+
+|                        |             |  Move the experimental  |              | Add symlink file to the    |
+| Experimental feature   | --------->  |  manifest file from     |  --------->  | promoted ops file in the   | 
+| is validated.          |             |  experimental folder to |              | operations folder to       |
+|                        |             |  the operations folder. |              | the experimental folder.   |
++------------------------+             +-------------------------+              +----------------------------+
+                                                                                            |
+                                                                                            |
+                                                    +---------------------------------------+
+                                                    |
+                                                    |
+                                                    |
+                                                    v
+                                        +------------------------+
+                                        |                        |
+                                        |   RelInt cuts a minor  |
+                                        |   release.             |
+                                        |         (•‿•)          |
+                                        +------------------------+
+```
+
+
+> It is critical that you do not remove the experimental ops file during your promotion.
+
+Operators might be still using the experimental ops file as part of their deployment and removing the file will sure break their environments. Once the deprecation notice is posted, the support for experimental ops file will be marked for removal on the next major cf-deployment release.
+
+## Deprecating an experimental ops file
+
+_Please do not remove the file or it's content._
+
+```
++------------------------+             +----------------------------+
+|                        |             |                            |
+| Experimental feature   | --------->  |  Add a deprecation notice  |
```

Comment:
"Add a deprecation notice to the ops file & readme."

Don't we actually want to....

"Replace the contents of the ops file with a deprecation notice (as a comment) and add deprecation notice to entry for that ops file in the readme."